Runtime/Core/ApiGatewayManagement/AnywhereGameServerAdapter.cs

```diff
@@ -0,0 +1,118 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Amazon.GameLift;
+using AmazonGameLiftPlugin.Core.ApiGatewayManagement.Models;
+using AmazonGameLiftPlugin.Core.Shared;
+using AmazonGameLiftPlugin.Core.Shared.Logging;
+using Amazon.GameLift.Model;
+
+namespace AmazonGameLiftPlugin.Core.ApiGatewayManagement 
+{
+    public class AnywhereGameServerAdapter : IGameServerAdapter
+    {
+        private readonly IAmazonGameLiftClientWrapper _amazonGameLiftClientWrapper;
+        private readonly string _fleetId;
+        private readonly string _fleetLocation;
+        private readonly string _playerIdPrefix = "playerId-";
+        
+        public AnywhereGameServerAdapter(IAmazonGameLiftClientWrapper amazonGameLiftClientWrapper, string fleetId, string fleetLocation )
+        {
+            _amazonGameLiftClientWrapper = amazonGameLiftClientWrapper;
+            _fleetLocation = fleetLocation;
+            _fleetId = fleetId;
```

Comment:
The rest of the changes look fine to me. One thing that does come to mind is again accessing this information since they should be available as configuration within Unity? We can discuss this in today's meeting